### PR TITLE
Fixed bug: Notebooks Vertical Scrollbar is Unnecessary for Some Grid Outputs

### PR DESCRIPTION
**Pull request by geleems**
This is for fixing bug: https://github.com/microsoft/azuredatastudio/issues/5836
It was because we did not count border pixels for the rows.
We need to add 1 pixel for each rows bottom border.
This code change fixes that.

Before fix:
image

After fix:
image

### CODE REVIEW

File: src/sql/workbench/parts/notebook/outputs/tableRenderers.ts [hunk 1]
```diff
@@ -67,7 +67,7 @@ export function renderDataResource(
 	detailTable.registerPlugin(new AdditionalKeyBindings());
 	let numRows = detailTable.grid.getDataLength();
 	// Need to include column headers and scrollbar, so that's why 1 needs to be added
-	let rowsHeight = (numRows + 1) * RESULTS_GRID_DEFAULTS.rowHeight + BOTTOM_PADDING_AND_SCROLLBAR;
```
Comment (chlafreniere) [inline src/sql/workbench/parts/notebook/outputs/tableRenderers.ts]: nit: don't need the extra parantheses

**Comment by chlafreniere** [comment]
Please double check that notebook grids look ok when many rows come back (maybe do a 'SELECT * FROM sys.all_objects'?)

File: src/sql/workbench/parts/notebook/outputs/tableRenderers.ts (hunk 1 above)
Comment (geleems) [reply src/sql/workbench/parts/notebook/outputs/tableRenderers.ts]: fixed.

**Comment by geleems** [comment]
It renders correctly with big table result (2397 rows)
```
select * from sys.all_objects;
```
image

**Pull request merged**